internal/pkg/agent/application/monitoring/server.go

```diff
@@ -75,7 +76,7 @@ func exposeMetricsEndpoint(
 }
 
 func createAgentMonitoringDrop(drop string) error {
-	if drop == "" || runtime.GOOS == "windows" {
+	if drop == "" || runtime.GOOS == "windows" || isHttpUrl(drop) {
```

Comment:
This looks like it fixes it, but why is an HTTP URL even being passed here?

Reply:
@blakerouse do you know the history on this? seems like the monitoring changed from http to the local socked between V1 and V2?